docs/qcqmc/experimental_wavefunctions.ipynb

```diff
@@ -0,0 +1,203 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Analyzing the Experimentally Reconstructed Wavefunctions\n",
+    "\n",
+    "The wavefunctions produced in the\n",
+    "[QCQMC](https://www.nature.com/articles/s41586-021-04351-z) paper are available\n",
+    "for download  from [zenodo](https://zenodo.org/records/10141262)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import urllib.request\n",
+    "import tarfile\n",
+    "\n",
+    "stream = urllib.request.urlopen(\"https://zenodo.org/records/10141262/files/qcqmc_data.tar.gz\")\n",
+    "\n",
+    "with tarfile.open(fileobj=stream, mode='r|gz') as tf:\n",
+    "    tf.extractall()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Look at the README from the downloaded data\n",
+    "from IPython.display import Markdown, display\n",
+    "display(Markdown('qcqmc_data/README.md'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading the wavefunctions\n",
+    "\n",
+    "Here we will reproduce some of the results from Section E of the SI from [QCQMC](https://www.nature.com/articles/s41586-021-04351-z).\n",
+    "\n",
+    "The variational energy of the trial wavefunction as a function of the number of cliffords is reported in Table S1, which we have reproduced here:\n",
+    "\n",
+    "| NCliffords | repeat 1 | repeat 2 | repeat 3 | repeat 4 |\n",
+    "|---|---|---|---|---|\n",
+    "| 10 | -1.800644 | -1.764747 | -1.813274 | -1.658202 |\n",
+    "| 16 | -1.823041 | -1.802192 | -1.840494 | -1.730591 |\n",
+    "| 28 | -1.906644 | -1.839835 | -1.843326 | -1.746749 |\n",
+    "| 47 | -1.925654 | -1.888527 | -1.860863 | -1.809656 |\n",
+    "| 80 | -1.909567 | -1.869456 | -1.887139 | -1.846339 |\n",
+    "| 136 | -1.930880 | -1.902309 | -1.889992 | -1.879164 |\n",
+    "| 229 | -1.944249 | -1.921523 | -1.903710 | -1.890947 |\n",
+    "| 387 | -1.947362 | -1.934682 | -1.910477 | -1.901883 |\n",
+    "| 652 | -1.952416 | -1.939853 | -1.912790 | -1.905250 |\n",
+    "| 1100 | -1.955544 | -1.944651 | -1.915073 | -1.909122 |\n",
+    "| 1856 | -1.955028 | -1.945966 | -1.909558 | -1.908038 |\n",
+    "| 3129 | -1.953877 | -1.947763 | -1.913386 | -1.908835 |\n",
+    "| 5276 | -1.954697 | -1.947323 | -1.912284 | -1.909315 |\n",
+    "| 8896 | -1.954930 | -1.947458 | -1.913889 | -1.913068 |\n",
+    "| 15000 | -1.954356 | -1.948894 | -1.913894 | -1.913082 |\n"
+   ]
+  },
```

Comment:
This notebook still has several code blocks in a row with almost no explanatory text.  It would be good if we can expound on what's going on a little bit more.